docs/guides/device-gateway/sdk.md

```diff
@@ -0,0 +1,344 @@
+---
+sidebar_position: 9
+title: Connect to Devices Using the Python SDK
+description: Connect to devices or your device gateway by embedding connectivity directly into your application with the Python SDK.
+tags:
+  - guides
+  - devices
+  - gateway
+  - security
+  - fleet
+  - iot
+  - sdk
+  - python
+---
+
+This guide provides step-by-step instructions for using ngrok as a device gateway.
+This example shows you how to embed connectivity into an application using ngrok's Python SDK to access
+an API running on an IoT device. All the Python code in this example is available in
+[this repo](https://github.com/ngrok-samples/ngrok-python-iot-agent).
```

Comment:
```suggestion
[the `ngrok-samples/ngrok-python-iot-agent` repo](https://github.com/ngrok-samples/ngrok-python-iot-agent).
```

I like being a bit more explicit with the link target, but take it or leave it!

Reply:
Somewhere along my journey, when submitting a PR for some docs, I was taught the same thing. I like the verbose link titles.

Reply:
Thank you! This is very helpful feedback. I didn't consider that.